src/collections/_documentation/workflow/releases/health.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Health (Beta)
+sidebar_order: 1
+---
+
+{% capture __alert_content -%}
+Sentry’s Release Health features are currently in beta. For more details about access to these features, feel free to reach out to mobile@sentry.io.
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+Monitor the health of releases by observing user adoption, usage of the application, percentage of [crashes](#crash), and [session data](#session). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+## How Release Health Works
+
+Once configured, Sentry's mobile and browser SDKs will send health data about the usage of your application.
```

Comment:
do we have release health on browser SDKs already? not sure about it.

Reply:
nope we need. to. remove the browser.

Reply:
I have removed it.